features/admin.actions.v1/components/action-endpoint-config-form.tsx

```diff
@@ -386,7 +387,7 @@ const ActionEndpointConfigForm: FunctionComponent<ActionEndpointConfigFormInterf
                             FormControlProps={ {
                                 margin: "dense"
                             } }
-                            ariaLabel="username"
+                            ariaLabel="authenticationType"
```

Comment:
Aria labels are supposed to be end user friendly.

```suggestion
                            ariaLabel="Authentication Type"
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7563/commits/7b54f298633dde6597c117612ecd1f4dd7c7232d